cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/Transformation.java

```diff
@@ -17,29 +17,19 @@
 package co.cask.cdap.templates.etl.api;
 
 /**
- * Properties that are required by a Stage.
+ * Transforms an input object into zero or more output objects.
```

Comment:
How can it be more than one? If you say split a string into array - it is still one object which is of type array right?